README.md

```diff
@@ -25,17 +42,29 @@ There are two types of selectors: recursive selectors and one-off selectors. Exa
 1. Recursive selector. This will install `truffle` package and clone/install all relevant dependencies, switching both `truffle` and all dependencies to the `develop` branch.
 
     ```
-    tc truffle:develop
+    tc use truffle:develop
     ```
 
-2. One-off selector. This will install the `truffle-contract` package and clone/all all relevant dependencies. Here, `truffle-contract` will be switched to the `json-schema` branch, but its dependencies will either be left alone (if already installed), or will default to the main branch of their repository, usually `master`.  
+2. One-off selector. This will install the `truffle-contract` package and clone/all all relevant dependencies. Here, `truffle-contract` will be switched to the `json-schema` branch, but its dependencies will either be left alone (if already installed), or will default to the main branch of their repository, usually `master`.
 
     ```
-    tc truffle-contract@json-schema
+    tc use truffle-contract@json-schema
     ```
-    
+
 You can also mix selectors. For instance, the following will install `truffle` and all of its dependencies at the `develop` branch, but leave `truffle-contract` at the `json-schema` branch:
 
 ```
-tc truffle:develop truffle-contract@json-schema
+tc use truffle:develop truffle-contract@json-schema
 ```
+
+## Reviewing Cross-Cutting Changes
+
+Imagine that someone sends a pull request to 3 separate truffle modules. You'll need to check out each branch in turn to set up your local environment in order to review their changes.
+
+Assuming that all of the branches have the same name, you can use
+
+```
+tc switch <org> <branch>
+```
+
+which will `git remote add`, `git fetch`, `git checkout`, `git pull`, and `git branch` for each module (if necessary/available).
```

Comment:
My first thought on this is that we can avoid this command simply by using `tc use truffle-core:develop truffle-migrate:remote/feature` and just do `git remote add` and `git fetch` under the hood.